MapboxNavigation/TopBannerViewController.swift

```diff
@@ -161,14 +161,17 @@ import MapboxDirections
         
         let controller = StepsViewController(routeProgress: progress)
         controller.delegate = self
+        let topHeight = view.bounds.height
+        let stepsTableExtent = UIScreen.main.bounds.height - topHeight
```

Comment:
Avoid using the bounds of the screen. It could be a split view or embedded navigation view. Can you access the NavigationViewController’s height?